### DISCUSSION TITLE
Optimal inference

### DISCUSSION THREAD

**Discussion by edmundbu** [microsoft/onnxruntime#22784] [Web Q&A]
Hello, I have a custom transformer model exported from PyTorch, and I am trying to deploy as a Chrome extension. For greedy/beam search, what is the best practice? I am in the process of using Javascript and ort.Tensor to create attention mask and input sequence at each step, but realized this could be a bit slow. Thanks!
